{"version":1,"ops":[{"type":5,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1553026751,"metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDIyMTQ5MDM3NTA="},"added":["bug"],"removed":[]},{"type":3,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1561387072,"metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDUwNTAwODE4Nw==","github-url":"https://github.com/perl11/cperl/issues/395#issuecomment-505008187"},"message":"With DEBUGGING at least I get a proper crash. Looking at it. Nice test case, thanks!","files":null},{"type":6,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1561387072,"metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdDI0MDI3NTg0NQ=="},"target":"4b284775ab6af3894050d3876340a928c9fe369d5774d6fc4c81c2dbb342024a","message":"With DEBUGGING at least I get a proper crash. Looking at it. Nice test case, thanks!\nThe problem is that the \\$buf arg is freed in leavesub, leave_scope, and not properly assigned back to the original value as planned.","files":null},{"type":6,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1561387094,"metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdDI0MDI3NjAyMg=="},"target":"4b284775ab6af3894050d3876340a928c9fe369d5774d6fc4c81c2dbb342024a","message":"With DEBUGGING at least I get a proper crash. Looking at it. Nice test case, thanks!\nThe problem is that the `\\$buf` arg is freed in leavesub, leave_scope, and not properly assigned back to the original value as planned.","files":null},{"type":3,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1562151571,"metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDUwODA0NjAzNg==","github-url":"https://github.com/perl11/cperl/issues/395#issuecomment-508046036"},"message":"Fixed with 5.30.0c, thanks for the nice test case.","files":null},{"type":4,"author":{"id":"04135dc995e2fdcd322aaf4268dbf04630a3cc7c"},"timestamp":1562151572,"metadata":{"github-id":"MDExOkNsb3NlZEV2ZW50MjQ1NzE3NDc5MA=="},"status":2}]}